packages/date-picker/docs/controls/validationControls.ts

```diff
@@ -0,0 +1,113 @@
+export const validationControls = {
```

Comment:
I'm still not a fan of being able to choose incompatible combinations using the controls, especially since there isn't any description or anything telling me I shouldn't (and it renders unexpectedly too, showing the start date having the error border, but rendering only the end date message).

<img width="1287" alt="image" src="https://user-images.githubusercontent.com/25891850/221453251-939b0851-4cd9-440e-b548-47f78d0c1307.png">

But if it's going to be merged in as is and fixed later, I think you also need to add these controls to DateRangeInputField.

Reply:
@HeartSquared I have updated both FilterDRP and the Input stories :)